css-syntax/styles.css

```diff
@@ -6,7 +6,8 @@ h1{
   text-align: center;
 }
 article{
-  padding left: 16px 0;
+  padding left: 16px;
```

Comment:
This is still incorrect. You have a space between `padding` and `left`, there are no spaces in property names. You want to use a `-`